### PR TITLE
enh(release): Add Centreon MBI 20.10.1 RN

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target serie

- [ ] 20.04.x
- [x] 20.10.x
- [ ] 21.04.x (master)

### CODE REVIEW

File: en/releases/centreon-commercial-extensions.md [hunk 1]
```diff
@@ -125,6 +125,14 @@ To use it, it is necessary to install and activate it. Refer to the
 
 ## Centreon MBI
 
+### 20.10.1
+
+#### Bugfixes
+
+- Update column length for BAM reporting
+- Inconsistency between Centreon & Centreon MBI column size for metric unit
+- Publication rule don't save backslash into CFIS publication
```
Comment (thiuyendang) [inline en/releases/centreon-commercial-extensions.md:134]: ..doesn't save

File: fr/releases/centreon-commercial-extensions.md [hunk 2]
```diff
@@ -133,6 +133,14 @@ Référez-vous à la [documentation dédiée](../graph-views/install.html#centre
 
 ## Centreon MBI
 
+### 20.10.1
+
+#### Correctifs
+
+- Mise à jour de la longueur de la colonne en base de données pour les rapports BAM
+- Incohérence entre la taille des colonnes Centreon et Centreon MBI pour l'unité métrique
```
Comment (thiuyendang) [inline fr/releases/centreon-commercial-extensions.md:141]: ..l'unité de la métrique

**Review by thiuyendang — CHANGES_REQUESTED** [review]

File: en/releases/centreon-commercial-extensions.md [hunk 3]
```diff
@@ -125,6 +125,14 @@ To use it, it is necessary to install and activate it. Refer to the
 
 ## Centreon MBI
 
+### 20.10.1
+
+#### Bugfixes
+
+- Update column length for BAM reporting
+- Inconsistency between Centreon & Centreon MBI column size for metric unit
```
Comment (thiuyendang) [inline en/releases/centreon-commercial-extensions.md:133]: "metric_unit"

**Review by thiuyendang — CHANGES_REQUESTED** [review]

**Review by thiuyendang — APPROVED** [review]

File: en/releases/centreon-commercial-extensions.md [hunk 4]
```diff
@@ -125,6 +125,14 @@ To use it, it is necessary to install and activate it. Refer to the
 
 ## Centreon MBI
 
+### 20.10.1
+
+#### Bugfixes
+
+- Update column length for BAM reporting
+- Inconsistency between Centreon & Centreon MBI column size for metric_unit
+- Publication rule doesn't save backslash into CFIS publication
```
Comment (tuntoja) [inline en/releases/centreon-commercial-extensions.md:134]:
```suggestion
- Publication rule doesn't save backslash into CIFS publication
```

File: fr/releases/centreon-commercial-extensions.md [hunk 5]
```diff
@@ -133,6 +133,14 @@ Référez-vous à la [documentation dédiée](../graph-views/install.html#centre
 
 ## Centreon MBI
 
+### 20.10.1
+
+#### Correctifs
+
+- Mise à jour de la longueur de la colonne en base de données pour les rapports BAM
+- Incohérence entre la taille des colonnes Centreon et Centreon MBI pour la colonne metric_unit
+- La règle de publication ne sauvegarde pas la barre oblique inverse dans la publication du CFIS
```
Comment (tuntoja) [inline fr/releases/centreon-commercial-extensions.md:142]:
```suggestion
- La règle de publication ne sauvegarde pas la barre oblique inverse dans la publication du CIFS
```

File: en/releases/centreon-commercial-extensions.md [hunk 6]
```diff
@@ -125,6 +125,14 @@ To use it, it is necessary to install and activate it. Refer to the
 
 ## Centreon MBI
 
+### 20.10.1
+
```
Comment (tuntoja) [inline en/releases/centreon-commercial-extensions.md:129]:
```suggestion

`April 29, 2021`

```

File: fr/releases/centreon-commercial-extensions.md [hunk 7]
```diff
@@ -133,6 +133,14 @@ Référez-vous à la [documentation dédiée](../graph-views/install.html#centre
 
 ## Centreon MBI
 
+### 20.10.1
+
```
Comment (tuntoja) [inline fr/releases/centreon-commercial-extensions.md:137]:
```suggestion

`29 avril 2021`

```

**Review by tuntoja — CHANGES_REQUESTED** [review]

**Review by tuntoja — APPROVED** [review]

**Review by thiuyendang — APPROVED** [review]

**Pull request merged**